views/js/qtiCreator/widgets/static/text/states/Active.js

```diff
@@ -1,27 +1,29 @@
 define([
+    'util/typeCaster',
     'taoQtiItem/qtiCreator/widgets/states/factory',
     'taoQtiItem/qtiCreator/widgets/static/states/Active',
     'taoQtiItem/qtiCreator/editor/ckEditor/htmlEditor',
     'taoQtiItem/qtiCreator/editor/gridEditor/content',
     'taoQtiItem/qtiCreator/widgets/helpers/formElement',
-    'tpl!taoQtiItem/qtiCreator/tpl/forms/static/text'
-], function(stateFactory, Active, htmlEditor, content, formElement, formTpl){
+    'tpl!taoQtiItem/qtiCreator/tpl/forms/static/text',
+    'taoQtiItem/qtiCreator/widgets/static/helpers/itemScrollingMethods'
+], function (typeCaster, stateFactory, Active, htmlEditor, content, formElement, formTpl, itemScrollingMethods) {
     'use strict';
 
-    var wrapperCls = 'custom-text-box';
+    var wrapperCls = 'text-block-wrap';
```

Comment:
Here will be regression. In prev version user created text block with class
After update it can't remove or update class
![image](https://user-images.githubusercontent.com/25976342/98252441-00d7f400-1f8b-11eb-9117-b17c647b1985.png)

Reply:
Done